lib/reference/keywords.mjs

```diff
@@ -296,7 +296,49 @@ export const namedTimelineRangeKeywords = new Set([
 	'exit-crossing',
 ]);
 
-export const systemColorsKeywords = new Set([
+const prefixedSystemColorKeywords = new Set([
```

Comment:
[suggestion] Can you add a code comment with links to web pages to maintain this list?
E.g. https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions#color_keywords

Reply:
```suggestion
// https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions#color_keywords
const prefixedSystemColorKeywords = new Set([
```